README.md

```diff
@@ -10,7 +10,8 @@ Docker-compose setup for experimental commons, small commons, or local developme
 
 
 ## Introduction
-This setup uses Docker containers for postgres, indexd, fence, peregrine, sheepdog, data-portal and nginx. Images for the cdis microservices and nginx will be pulled from quay.io (master), while postgres (9.5) images will be pulled from Docker Hub. Nginx will be used as a reverse proxy to each of the services. Below you will find information about [migrating existing](#Release-History-and-Migration-Instructions) and [setting up](#Setup) new compose services, some [tips](#Dev-Tips) and basic information about [using](#Using-the-data-commons) data commons. You can quickly find commonly used commands in our [cheat sheet](./docs/cheat_sheet.md). Config file formats were copied from [cloud-automation](https://github.com/uc-cdis/cloud-automation) and stored in the `Secrets` directory and modified for local use with Docker Compose. Setup scripts for some of the containers are kept in the `scripts` directory.
+=======
```

Comment:
I don't think this is what you were going for:
<img width="913" alt="Screen Shot 2019-03-25 at 12 05 04 PM" src="https://user-images.githubusercontent.com/4224001/54939333-55357980-4ef6-11e9-891b-f1e89495d003.png">

Reply:
Ouch, thanks.